mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -494,6 +702,14 @@ static LogicalResult populateHostHarnessLogic(ModuleOp &module, OpBuilder &build
     block->push_back(printMemRefCallOp);
   }
 
+  // Populate host validation logic
+  if (populateValidation.getValue()) {
+    populateHostValidationLogic(
+        module, builder, context, dataType, block, ref2MemsetValue,
+        mcpuMemset4DFuncOp, filterDimension, inputDimension, outputDimension,
+        filterHostAllocOp, inputHostAllocOp, outputHostAllocOp);
+  }
+
```

Comment:
Can you decouple the host validation from host harness? They should stay as two independent process. Use case wise, a user might want only the host harness but not validation code.

Reply:
The user can use -ph to only generate the host harness, and -pv to generate validation code